Makefile

```diff
@@ -40,11 +40,18 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v
 test-e2e: test-unit
 
-update-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api
-update-codegen: update-codegen-crds
-verify-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api --verify-only
-verify-codegen: verify-codegen-crds
+PKG := ${PWD:${GOPATH}/%=%}
+crd-schema-gen:
+	git clone -b release-4.2 --single-branch --depth 1 https://github.com/openshift/crd-schema-gen.git $(TMP_GOPATH)/src/github.com/openshift/crd-schema-gen
+	GOPATH=$(TMP_GOPATH) go install $(TMP_GOPATH)/src/github.com/openshift/crd-schema-gen/cmd/crd-schema-gen
+	crd-schema-gen --apis-dir vendor/github.com/openshift/api/operator/v1
```

Comment:
by convention we end phony target with `.PHONY: target-name

Reply:
I see that was preexisting, so it's up to you to fix it

Reply:
Is adding it to the .PHONY list at the end good?

Reply:
it works the same way, our convention is to use it as `{}` to scope the target